### PR TITLE
feat(C#): number-sort.cs

### PR DESCRIPTION
**Pull request by xHyroM**

### CODE REVIEW

File: C#/number-sort.cs [hunk 1]
```diff
@@ -2,7 +2,7 @@ public class Sort
 {
     public static void Main()
     {
-        int[] myArr= new int[5] {1,5,3,8,21};
+        int[] myArr= new int[5] {1,9,728,320,123,845,21};
```
Comment (rhydderchc) [inline C#/number-sort.cs:5]: You've initialized the size of the array as `5`, but the number of elements in the array are more than five, it breaks the program.

**Review by rhydderchc — CHANGES_REQUESTED** [review]

File: C#/number-sort.cs [hunk 2]
```diff
@@ -2,7 +2,7 @@ public class Sort
 {
```
Comment (rhydderchc) [inline C#/number-sort.cs:2]: You must also import `System` or else it breaks the program, and Array and Console remain unidentified.

**Review by rhydderchc — CHANGES_REQUESTED** [review]

File: C#/number-sort.cs (hunk 1 above)
Comment (xHyroM) [reply C#/number-sort.cs:5]: Oh

**Comment by xHyroM** [comment]
Fixed

**Comment by rhydderchc** [comment]
Awesome 👍🏻

**Pull request merged**